helm/minio/values.yaml

```diff
@@ -308,6 +308,27 @@ makeBucketJob:
   tolerations: []
   affinity: {}
 
+## List of command to run after minio install
+## NOTE: the mc command TARGET is always "myminio"
+customCommands:
+  # - command: "admin policy set myminio consoleAdmin group='cn=ops,cn=groups,dc=example,dc=com'"
+
```

Comment:
This supposed is being added here https://github.com/minio/minio/pull/14216

Reply:
Thanks, I checked that PR, but it's unclear for me how that would help me to set an existing policy to an LDAP group as my example. 

I got what you say that this is a bit of a stretch to run commands like this, although I think it could be useful for several - otherwise not covered - use cases.